modules/programs/fish.nix

```diff
@@ -279,6 +289,15 @@ in {
       '';
     };
 
+    programs.fish.formatFishScripts = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Whether to process fish configuration and scripts with
+        <literal>fish_indent</literal>.
+      '';
+    };
```

Comment:
I initially tried to implement it like this, but that ended up making tests more complicated: so many tests already stub `fish`, and I had trouble making a “dummy” `fish_indent` to stub in.

Reply:
Well, since we now pull in Fish in the tests anyway I think we can remove those stubs 🙂

Would you mind having a look at "unstubbing" fish and removing this option?

Reply:
Happy to do it!

Reply:
https://github.com/nix-community/home-manager/pull/3470